src/app/regex-input/regex-input.component.css

```diff
@@ -1,12 +1,18 @@
+* {
```

Comment:
Is this required? can we use input selector or another individual component/class instead of * ?

Reply:
Yeah, I guess, I just needed al three to have that property. I could put it manually in all three I guess, or make another class